app/templates/people/index.hbs

```diff
@@ -4,29 +4,30 @@
       <div class="au-c-sidebar__content au-o-box">
         <form {{on "submit" this.search}} class="au-o-grid au-o-grid--small">
           <p class="au-o-grid__item au-u-1-2@small au-u-1-1@medium">
```

Comment:
I think the au-u-* classes can be removed. It looks a bit strange at the moment:

![image](https://user-images.githubusercontent.com/3533236/152343888-f9028bf4-24ea-4ed8-9212-a37db8acd206.png)